sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -845,8 +845,9 @@ object RapidsConf {
 
   val EXPLAIN = conf("spark.rapids.sql.explain")
     .doc("Explain why some parts of a query were not placed on a GPU or not. Possible " +
-      "values are ALL: print everything, NONE: print nothing, NOT_ON_GPU: print only parts of " +
-      "a query that did not go on the GPU")
+      "values are ALL_NO_CBO: print everything except cost-based optimizations, ALL: print " +
+        "everything including cost-based optimizations, NONE: print nothing, NOT_ON_GPU: print " +
```

Comment:
Seeing this I'm wondering whether it's worth it to have something like NOT_ON_GPU_NO_CBO which indicates we may want to control CBO output orthogonally via a separate config as @revans2 mentioned.

Reply:
I've reverted the changes to the `EXPLAIN` config and introduced a separate `spark.rapids.sql.optimizer.explain`. I think this makes more sense for now because all of the optimizer configs are internal so it seems unusual to update the documented `EXPLAIN` config to reference them.